README.md

```diff
@@ -204,6 +206,17 @@ Given that the instructions in the [download section](#download) have been follo
 ```
 where `<keypair_name>.sec.pem` the private key created in the [relevant section](#create-keys) and `<file_to_decrypt>` one of the files downloaded following the instructions of the [download section](#download-file).
 
+## Login
+
+You can login to download the configuration file needed for some of the the tool's operation using the login command:
+```bash
+./sda-cli login <login_target>
+```
+where `login_target` is the URL for the the RI Life Science login, or for Big Picture: https://login.bp.nbis.se/
```

Comment:
There is an inconsistency here and in the argHelp (`login.go` line 34) where the url is `bp.nbis.se` instead. Also, the sentence is not exactly correct, so I am changing it a bit:
```suggestion
where `login_target` is the URL can be the login endpoint for Big Picture (https://login.bp.nbis.se/), Federated EGA (https://login.test.fega.nbis.se/) or Genomic Data Infrastructure (https://login.gdi.nbis.se/)
```

Reply:
Good catch, updated it.